### DISCUSSION TITLE
SQL Server on any Mac Computers?

### DISCUSSION THREAD

**Discussion by melomelo926** [prisma/prisma#8419] [Q&A] [answered]
Is it a must to install Docker before installing the SQL Server on any Mac Computers?

**Comment by pantharshit00** [comment, marked as answer]
Yes, there is no other way. Before SQL Server 2017, you had to create a Windows VM to use SQL server on Mac. At least now it is available via Docker. 

You can refer to this dataguide article for installation instructions for different platforms: https://www.prisma.io/dataguide/mssql/setting-up-a-local-sql-server-database#setting-up-sql-server-on-macos

**Comment by melomelo926** [comment]
OK.  Do I need to have SQL server management to see the development, QA, and production environment?

**Comment by melomelo926** [comment]
I meant the contents/applications in the development, QA, and etc. production environment.

**Comment by melomelo926** [comment]
Without installing SQL server management, how can I view the content in those three environments?  What "impersonate user" utilities I can use?

**Reply by janpio** [reply]
To look at the database content, you can also use Prisma Studio.
(This has nothing to do with "impersonate user" though, so not sure what the connection there is)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
